app/views/admin/projects/_translatable.html.slim

```diff
@@ -2,7 +2,7 @@
   = f.input :summary, wrapper_html: {class: 'hidden-print'}
     = project.summary
   = f.input :details
-    = simple_format(project.details)
+    = simple_format(project.details.to_s)
```

Comment:
a while back i was wondering if there was a way to DRY up the .to_s's on translatables. . and failed. curious if you have any observations about how translatables is working & why? (in terms of software design?)

Reply:
Good question. I'll look into it and get back to you.